packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -51,28 +48,35 @@ class Product extends Component {
 
     render() {
         const { options, props } = this;
-        const { classes, item } = props;
-        const amount = item.price.regularPrice.amount;
+        const { classes, item, currencyCode } = props;
 
         return (
             <li className={classes.root}>
-                <div className={classes.image} />
+                <div
+                    className={classes.image}
+                    style={{
```

Comment:
Please define this style object outside of the return statement, either as a `const` or a getter.